WeddingSiteBuilder/Views/register.html

```diff
@@ -39,45 +39,30 @@
 
 
             var body = {};
-            body.fname = $('#firstname').val();
-            body.lname = $('#lastname').val();
-            body.email = $('#email').val();
-            body.pword = $('#password').val();
-            //body.who = $('').val();
+            body.FirstName = $('#firstname').val();
```

Comment:
Remember these variables must be named exactly like they are named in the Registration DTO and such on the C# side so it matters between email and Email and fname and FirstName